src/components/common/PrivateRoute.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+function PrivateRoute({ component: Component, isAuthenticated, ...rest }) {
+  return (
+    <Route
+      {...rest}
+      render={props =>
+        isAuthenticated ? (
```

Comment:
This is very hard to read. Please avoid ternary statements when both outcomes are components (the only exception is when it's super small) in general it's better to avoid them.

It would be much easier to read as:
```
  if (!isAuthenticated) {
     return <Redirect to={{ pathname: '/login' }} />;
  }

  return <Component {...props} />;
```